Sources/ProjectSpec/Scheme.swift

```diff
@@ -141,6 +146,7 @@ extension Scheme.Test: JSONObjectConvertible {
 
     public init(jsonDictionary: JSONDictionary) throws {
         config = try jsonDictionary.json(atKeyPath: "config")
+        codeCoverageEnabled = try jsonDictionary.json(atKeyPath: "codeCoverageEnabled")
```

Comment:
This will fail if `codeCoverageEnabled` isn't present
```
codeCoverageEnabled = jsonDictionary.json(atKeyPath: "codeCoverageEnabled") ?? false
```

Reply:
Thanks! I did it and removed it in my own code review... 😄 Fixing it...